Source/JavaScriptCore/llint/WebAssembly64.asm

```diff
@@ -213,6 +213,7 @@ wasmOp(i32_rem_s, WasmI32RemS, macro (ctx)
         subi t0, t2, t2
     elsif RISCV64
         remis t1, t0
+        move t0, t2
```

Comment:
I think one lowering pass expands `remis t1, t0` into `remis t1, t0, t0`, so the extra move might be avoidable by just explicitly listing `t2` as the destination register in `remis` and friends.

Reply:
I tried that initially and there was a bit more to it than that, but I can certainly revisit this solution to resolve things in the platform-specific file.

Reply:
It's not a separate pass, it's a detail of `riscv64LowerOperation` and specifically handling of the `rem*` group of instructions inside `emitMultiplicationOperation`. For `remis t1, t0`, with `[t1, t0]` being the initial operands, the operands array should get replaced with `[t0, t1, t0]`. So a move-avoiding `remis` would need `[t0, t1, t2]` as operands, IIUC.